hydroshare/settings.desired

```diff
@@ -719,22 +719,6 @@ SWAGGER_SETTINGS = {
 # detect test mode to turn off some features
 TESTING = len(sys.argv) > 1 and sys.argv[1] == 'test'
 
-REDIS_HOST = os.environ.get('REDIS_PORT_6379_TCP_ADDR', 'localhost')
-REDIS_PORT = 6379
-
-REDIS_CONNECTION = redis.Redis(
-    host=REDIS_HOST,
-    port=REDIS_PORT,
-    db=4)
-WMS_CACHE_DB = redis.Redis(
-    host=REDIS_HOST,
-    port=REDIS_PORT,
-    db=5)
-PERMISSIONS_DB = redis.Redis(
-    host=REDIS_HOST,
-    port=REDIS_PORT,
-    db=6)
-
```

Comment:
@cuahsimarko Should these REDIS related settings be removed from settings.py as well?

Reply:
For this version of remove-redis (with the dev pull) none such settings exist in settings.py

Reply:
@cuahsimarko Oh, so this settings.desired is only used for alternative development environment setup? I am confused what this settings.desired file is for.

Reply:
Currently, it's not used anywhere. Right now, if there is not a local_settings.py (in whatever environment), the build will fail. Eventually, settings.desired should be evaluated, then replace settings.py so that the app will technically launch, without a local_settings.py. This was problematic, with all the other changes going on, so a bookmark of all the time I spend on settings.py is preserved in settings.desired